src/pkg/cli/client/byoc.go

```diff
@@ -42,7 +43,7 @@ const (
 
 var (
 	// Changing this will cause issues if two clients with different versions are using the same account
-	cdImage = pkg.Getenv("DEFANG_CD_IMAGE", "public.ecr.aws/defang-io/cd:beta")
+	cdImage = pkg.Getenv("DEFANG_CD_IMAGE", "public.ecr.aws/defang-io/cd:public-beta")
```

Comment:
?? we haven't changed the GitHub Action that copies the image there. Also, that repo should only ever have public images, so the extra "public-" seems superfluous.

Reply:
It is mainly to break the dependency loop between cli and different CD version, preventing older cli version from using the new CD image which will be missing pulumi, or the new cli using the older CD image which does not expose the right volumes.